libbpf-tools/bitesize.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __BITESIZE_H
+#define __BITESIZE_H
+
+#define TASK_COMM_LEN	16
+#define MAX_SLOTS	36
```

Comment:
2^36 is a huge block size, do we really need so many? 30 (up to 1GB) should be plenty, no?

UPD: oh, just noticed that you are counting in KBs, so subtract 10 from that: 20 should be plenty.

Reply:
Oh sorry for this, I forgot to recalculate this value, although you mentioned it in the last pr, I'll fix it and keep this in mind. Thank you.